CHANGES.txt

```diff
@@ -5,6 +5,10 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Remove limitation that prevented usage of columns from multiple tables
+   in the same order by expression.
+   eg: select t1.x, t2.y, t3.y from t1, t2, t3 order by t1.x - t2.y + t3.z
```

Comment:
afaik an order by example without select and from is enough, also maybe use capital letters